Cargo.toml

```diff
@@ -16,6 +16,13 @@ exclude = [
   'openssl/test/*',
 ]
 
+[features]
+default = []
+weak-crypto = []
+camellia = []
+idea = []
+seed = []
```

Comment:
Could you add comments to each of these features indicating what they enable? (I'm not sure what "camellia", "seed", or "idea" are)

Reply:
Added in Cargo.toml, but maybe these features should be documented in the README as well?